security-compatibility-with-mysql.md

```diff
@@ -140,3 +140,113 @@ The support for TLS authentication is configured differently. For detailed infor
 | ed25519 (MariaDB)            | No               |
 | GSSAPI (MariaDB)             | No               |
 | FIDO                         | No               |
+
+### `tidb_auth_token`
+
+`tidb_auth_token` is a passwordless authentication method based on [JSON Web Token (JWT)](https://datatracker.ietf.org/doc/html/rfc7519). In v6.4.0, `tidb_auth_token` is only used for user authentication in TiDB Cloud. Starting from v6.5.0, you can also configure `tidb_auth_token` as a user authentication method for TiDB Self-Hosted. Different from password-based authentication methods such as `mysql_native_passsword` and `caching_sha2_password`, when creating users using `tidb_auth_token`, there is no need to set or store custom passwords. To log into TiDB, users only need to use a signed token instead of a password, which simplifies the authentication process and improves security.
+
+JWT consists of 3 parts: Header, Payload, and Signature. After being encoded using base64, they are concatenated into a stirng separated by dots (`.`).
+
+The Header describe the meta data of the JWT, including 3 parameters:
+
+* `alg` means the algorithm for signature, default as `RS256`
+* `typ` means the type of token, unified as `JWT`
+* `kid` means the key id for generating token signature
+
+Here is an example for Header:
+
+```json
+{
+  "alg": "RS256",
+  "kid": "the-key-id-0",
+  "typ": "JWT"
+}
+```
+
+The Payload is the main part of JWT, which stores the user information in *clains*. These claims are required by `tidb_auth_token` users:
+
+* `iss`: if `TOKEN_ISSUER` is not set or set to empty when [`CREATE USER`](/sql-statements/sql-statement-create-user.md), this claim is not required; otherwise this claim should be the same as the setting value
+* `sub`: this claim is required to be the same as the user name
+* `iat`: means `issued at`. In TiDB, it is required not to be later than the time of authentication neither earlier than 15 minutes before authentication
+* `exp`: means `expiration time`. If it is earlier than the time of authentication, the authentication fails
+* `email`: the email can be specified when creating user by `ATTRIBUTE '{"email": "xxxx@pingcap.com"}`. If no email was giving when creating user, this claim should be set as an empty string; or it should be the same as the specified value
```

Comment:
```suggestion
* `email`: the email can be specified when creating a user by `ATTRIBUTE '{"email": "xxxx@pingcap.com"}`. If no email is specified when creating a user, this claim should be set as an empty string; otherwise, it should be the same as the specified value.
```